pkg/app/app.go

```diff
@@ -97,6 +97,7 @@ func New() EnvdApp {
 		CommandDebug,
 		CommandVersion,
 		CommandTop,
+		CommandDemo,
```

Comment:
How about reusing `envd init`?

Reply:
I am fine with what you decide, I created a new command just in case.